### PR TITLE
Virology rebalance

### PR DESCRIPTION
**Pull request by GuyonBroadway**
:cl: GuyonBroadway
balance: Virology can now have up to three viruses at once, just like we used to!
/:cl:

[why]: # People never play viro these days as the symptoms after all their buffs and nerfs are hardly worth it. Even with the stats based effectiveness and the level 7 and 8 symptoms its still not very good. I changed this back to how it used to work.

### CODE REVIEW

**Comment by GuyonBroadway** [comment]
All is tested, checked and runtime free.

File: hippiestation/code/datums/diseases/_MobProcs.dm [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	if(D.GetDiseaseID() in resistances)
+		return FALSE
+
+	if(HasDisease(D))
+		return FALSE
+
+	if(!(type in D.viable_mobtypes))
+		return FALSE
+
+	if(count_by_type(viruses, /datum/disease/advance) >= 3)
```
Comment (KayeArray) [inline hippiestation/code/datums/diseases/_MobProcs.dm]: What's the need for mirroring the other proc if you're only changing this line?

**Also:**
If you need to mirror a file, or function into a hippie-specific file, please leave behind a comment stating where it went.

// hippie start - Mirrored this function in <file> for <reason>
bunch of shitcode here
// hippie stop
Once you mirror a file, please follow the above for marking your changes, this way we know what needs to be updated when a file has been mirrored.

Comment (GuyonBroadway) [reply hippiestation/code/datums/diseases/_MobProcs.dm]: its adding more than one line but I'll add the shitcomment in.

**Comment by KayeArray** [comment]
Does this also take into account the fact that viruses were changed to override each other based on resistance I believe?

**Comment by GuyonBroadway** [comment]
Yes I literally used a proc override to undo that.

**Comment by GuyonBroadway** [comment]
And tested it in game, gave myself three different viruses to check and they all worked.

**Comment by tortellinitony** [comment]
we should definitely have documentation not in the original files

**Comment by GuyonBroadway** [comment]
I hate it when my PR's get caught up in between a maintainer tussle....

**Comment by dreamcrab** [comment]
im stupid what does this mean for the layman

**Comment by GuyonBroadway** [comment]
Time was you could have three independent viruses in your system at once, this meant that you could have three virus with a single very potent effect as opposed to one with several medium effects.

A skilled virologist could make the crew regen so much you had to kill them dead dead else they would get outta crit and be able to stagger home.

Due to the RNG nature of viro this was kinda hard to do. and each virus has to be different so the symptoms had to be swapped.

File: hippiestation/code/datums/diseases/_MobProcs.dm (hunk 1 above)
Comment (ghost) [reply hippiestation/code/datums/diseases/_MobProcs.dm]: You've made changes to /tg/ files, please mark these changes.

**All modifications to non-hippie files should be marked.**

- Multi line changes start with `// hippie start` and end with `// hippie end`
- You can put a messages with a change if it isn't obvious, like this: `// hippie start - reason`
  - Should generally be about the reason the change was made, what it was before, or what the change is
  - Multi-line messages should start with `// hippie start` and use `/* Multi line message here */` for the message itself
- Single line changes should have `// hippie` or `// hippie - reason`

If you need to mirror a file, or function into a hippie-specific file, please leave behind a comment stating where it went.

```
// hippie start - Mirrored this function in <file> for <reason>
bunch of shitcode here
// hippie stop
```

Once you mirror a file, please follow the above for marking your changes, this way we know what needs to be updated when a file has been mirrored.


The last part here is very important

**Comment by GuyonBroadway** [comment]
MRTY must've added my pr to the list of prs the upstream merge closes by mistake.

**Comment by ghost** [comment]
yes

**Comment by GuyonBroadway** [comment]
Ok this is all commented up.

**Review by KayeArray — APPROVED** [review]

**Comment by KayeArray** [comment]
+1.00

**Comment by ghost** [comment]
this is fine

**Pull request merged**